lib/init/grass.py

```diff
@@ -379,50 +381,34 @@ def help_message(default_gui):
     sys.stderr.write(s)
 
 
-def get_grass_config_dir():
-    """Get configuration directory
+def create_grass_config_dir():
+    """Create configuration directory
```

Comment:
Without examining the code more closely, I don't see where the creation happens, but it is not here. I think "creation" is expected to refer to creation of the actual directory rather than construction of the path which is referred to by the simple "get".

Reply:
Creation happens here: https://github.com/imincik/grass/blob/config-dir-env-variable/lib/init/grass.py#L404